recipes/datrie/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "datrie" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a11371cc2dbbad71d6dfef57ced6e8b384bb377eeb847c63d58f8dc8e8b2023
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: 
+    - cython src/datrie.pyx
+    - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed  -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython
+    - pytest-runner
+    - setuptools
+  run:
+    - python    
+
+test:
+  imports:
+    - datrie
+
+about:
+  home: https://github.com/kmike/datrie/
+  license: LGPLv2
+  license_family: LGPL
+  license_file: COPYING
+  summary: Super-fast, efficiently stored Trie for Python, uses libdatrie
+  doc_url: https://pypi.python.org/pypi/datrie/0.1.1
```

Comment:
Should we drop the version from here? Also is this what they recommend for docs?

Reply:
Yes. I will remove it.

Reply:
Thanks :)